src/renderer/components/SelectMediaBlock.tsx

```diff
@@ -1,10 +1,17 @@
-import { Box, colors, styled, Typography } from '@mui/material';
-import AddIcon from '@mui/icons-material/Add';
+import { Box, styled, Typography, Stack } from '@mui/material';
 import { Dispatch, SetStateAction } from 'react';
+import colors from 'renderer/colors';
+import { extractFileNameWithExtension } from 'renderer/util';
+import ActionButton from './ActionButton';
 
 const SelectMediaBox = styled(Box)`
-  background: ${colors.grey[400]};
-  color: ${colors.grey[900]};
+  width: 100%;
+`;
+
+const InnerBox = styled(Box)`
+  border-style: dashed;
+  border-color: ${colors.grey[500]}
```

Comment:
@hhen0005 / @fwang1703 / someone experienced with front end - the border colour is set to grey 500.... so why does it show up as black in the app?

Reply:
MUI really do be like that

Reply:
Missing semicolon on line 13, this is now fixed